backend/src/test/java/com/daedan/festabook/festival/controller/FestivalControllerTest.java

```diff
@@ -382,12 +397,15 @@ class removeFestivalImage {
             Festival festival = FestivalFixture.create();
             festivalJpaRepository.save(festival);
 
+            String token = jwtTestHelper.createCouncilAndLogin(festival);
+
             FestivalImage festivalImage = FestivalImageFixture.create(festival, 1);
             festivalImageJpaRepository.save(festivalImage);
 
             // when & then
             RestAssured
                     .given()
+                    .header(AUTHENTICATION_HEADER, AUTHENTICATION_SCHEME + token)
                     .header(FESTIVAL_HEADER_NAME, festival.getId())
                     .contentType(ContentType.JSON)
                     .when()
```

Comment:
```suggestion
            Header authorizationHeader = jwtTestHelper.createCouncilAndLogin(festival);

            FestivalImage festivalImage = FestivalImageFixture.create(festival, 1);
            festivalImageJpaRepository.save(festivalImage);

            // when & then
            RestAssured
                    .given()
                    .header(authorizationHeader)
                    .header(FESTIVAL_HEADER_NAME, festival.getId())
                    .contentType(ContentType.JSON)
                    .when()
```
의견인데요
토큰 말고, Header 객체(RestAssured의 header()에 들어가는 인자)를 생성해서, 사용하도록 묶어내면 반복적으로 나오는 정적 변수들 (AUTHENTICATION_HEADER, AUTHENTICATION_SCHEME)을 제거할 수 있을 것 같은데 어떤가요?

Reply:
천..천재? 반복적이지만 반영해볼게요...